engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -1148,40 +1148,40 @@ void Myst::o_bookGivePage(uint16 var, const ArgumentsArray &args) {
 	uint16 mask = 0;
 
 	switch (_globals.heldPage) {
```

Comment:
This now needs an empty default case or will otherwise generate a compiler warning because of unhandled enum values.

Reply:
Indeed, maybe L1142-44 should be merged into the switch.